src/functionalTest/java/uk/gov/hmcts/probate/functional/utils/TestUtils.java

```diff
@@ -29,23 +54,21 @@ public String getJsonFromFile(String fileName) {
         }
     }
 
-    public Headers getHeaders(String sessionId) {
-        return Headers.headers(
-            new Header("Content-Type", ContentType.JSON.toString()),
-            new Header("Session-ID", sessionId));
+    public Headers getCitizenHeaders() {
+        return getHeaders(citizenEmail);
     }
 
-    public Headers submitHeaders(String sessionId) {
-        return Headers.headers(
-            new Header("Content-Type", ContentType.JSON.toString()),
-            new Header("UserId", sessionId),
-            new Header("Authorization", "DUMMY_KEY"));
+    public Headers getCaseworkerHeaders() {
```

Comment:
Is this used?

Reply:
currently not used but going forward for other request we will be using it.